pkg/proc/variables.go

```diff
@@ -734,6 +734,11 @@ func resolveTypedef(typ godwarf.Type) godwarf.Type {
 	}
 }
 
+// SetMemory sets the memory for the variable.
+func (v *Variable) SetMemory(mem MemoryReadWriter) {
```

Comment:
Is this a leftover? I can't find where it's called.

Reply:
It is, removed.